releasenotes/notes/add-heat-experiment-047a73818407e733.yaml

```diff
@@ -0,0 +1,24 @@
+---
+features:
+  - |
+    New experiment module :mod:`qiskit_experiments.library.hamiltonian`
+    has been added. This module collects experiments to characterize low level
+    system properties, such as device Hamiltonian.
```

Comment:
I am not sure about this. I think this description applies more to experiments like `CrossResonanceHamiltonian` or a ZZ experiment (which does not exist). The HEAT experiments here are focused more on characterizing the given pulse schedule which could be tweaked to try to minimize deviations from ZX than on characterizing the system.

Reply:
This is good point. Another option would be removing dedicated module. Any thoughts?

Reply:
I don't have a strong opinion. On the one hand, I don't see what it is doing as much different from what `fine_amplitude.py` is doing other than doing a two qubit operation instead of one qubit. On the other hand, quantum volume, randomized benchmarking and tomography (two other subpackages of library) are also forms of characterization, so the divisions are a little arbitrary. I think I would keep cr_hamiltonian.py and heat together though. They are kind of rough and fine equivalents of the same experiment.